neuralpy/layers/bilinear.py

```diff
@@ -0,0 +1,107 @@
+"""Dense layer for NeuralPy"""
```

Comment:
Can you update this line from "Dense layer for NeuralPy" to  "Bilinear layer for NeuralPy"?